Lua/Lua.sublime-syntax

```diff
@@ -303,6 +304,20 @@ contexts:
 
     - include: else-pop
 
+  variable-modifier:
+    - match: \<(?=\b(?:const|close)\b)
```

Comment:
Why escape `<`? The `\b` seems very redundant, seeing you match on every character that comes before or after. 

Also doesn't allow whitespace before the attribute, which is legal.
```
> do local t < const > = 2; print(t); end
2
```

Since you're matching almost the entire token in the lookahead already, you might as well match the entire thing in a single expression. It's not like multiple attributes are allowed either.

Reply:
I just checked on the Lua online demo, and it looks like there can be a newline between the `<` and the modifier:

```lua
local text <
const > = "Hello world!"

io.write(text)
```

Is the lookahead required at all? Could the `<` in `local text <` ever be anything else? If so, it might be necessary to use a branch rather than a lookahead.

Reply:
> Either way, you should add a test case that a missing > recovers gracefully

`else-pop` to the rescur :) I forgot to add that so that missing `>` doesn't break the highlighting.